pkg/api/meta/meta.go

```diff
@@ -506,8 +548,20 @@ func (a genericAccessor) SetAnnotations(annotations map[string]string) {
 	*a.annotations = annotations
 }
 
-func (a genericAccessor) GetOwnerReferences() []unversioned.ObjectReference {
-	return a.ownerReferences
+func (a genericAccessor) GetOwnerReferences() []OwnerRefereceAccessor {
```

Comment:
I think `GetOwnerReferences() []OwnerReferences` is more user friendly than `GetOwnerReferences(i int) OwnerRefereceAccessor`.

And `AppendOwnerReference() OwnerReferenceAccessor` is more useful than `InsertOwnerReference(i int) OwnerReferenceAccessor`, because the order doesn't matter.

The standard object meta can implement these two methods easily.

What do you think?